Empire/EmpireManager.cpp

```diff
@@ -46,6 +46,11 @@ const Empire* EmpireManager::GetEmpire(int id) const {
     return it == m_empire_map.end() ? nullptr : it->second;
 }
 
+std::shared_ptr<const UniverseObject> EmpireManager::GetSource(int id) const {
+    auto it{m_empire_map.find(id)};
```

Comment:
The OSX clang compiler spitted out error messages the last time I used a similar construct.

See 89f801cc3e